config/initializers/filter_parameter_logging.rb

```diff
@@ -3,4 +3,6 @@
 # Be sure to restart your server when you modify this file.
 
 # Configure sensitive parameters which will be filtered from the log file.
-Rails.application.config.filter_parameters += [:password]
+Rails.application.config.filter_parameters += [
+  :passw, :secret, :token, :_key, :crypt, :salt, :certificate, :otp, :ssn
```

Comment:
`passw` ? I guess it's a glob/wildcard match?

Reply:
i just took what came as part of Rails 6.1.  Do you think I should add `:password` just in case?

Reply:
@ndushay Nah, I think going with the Rails defaults makes good sense.